.github/workflows/smart-integ-inferno-test-periodically.yaml

```diff
@@ -132,7 +130,6 @@ jobs:
     needs: [inferno-test, custom-integration-tests]
```

Comment:
Should we remove the checks on these dependency workflows as well?

Reply:
Yes because once the dependency is met it should run. We wouldnt want to wait for someone to approve